feature_extraction.py

```diff
@@ -140,3 +140,215 @@ def transform(self, X):
                         how='left')
 
         return {'numerical_features': X[self.groupby_aggregations_names].astype(np.float32)}
+
+
+class Application(BaseTransformer):
```

Comment:
@pknut this name is not very descriptive. What is the operation (logic) that this transformer performs?